frontend/components/NavigationBar.vue

```diff
@@ -1,85 +1,136 @@
 <template>
-  <Disclosure as="nav" class="bg-primary" v-slot="{ open }">
-    <div class="max-w-7xl ml-2 mr-auto px-2 sm:px-6 lg:px-8 h-10">
-      <div class="absolute inset-y-0 left-0 flex items-center sm:hidden">
-        <DisclosureButton
-          class="inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
-        >
-          <span class="sr-only">Open main menu</span>
-          <MenuIcon v-if="!open" class="block h-6 w-6" aria-hidden="true" />
-          <XIcon v-else class="block h-6 w-6" aria-hidden="true" />
-        </DisclosureButton>
-      </div>
-      <div
-        class="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start"
-      >
-        <div class="flex-shrink-0 flex items-center">
-          <img
-            class="lg:block h-10 w-auto"
-            src="../assets/images/navbar_logo.svg"
-          />
+  <Disclosure as="nav" class="bg-primary z-50" v-slot="{ open }">
+    <div class="mx-auto px-4 sm:px-6 lg:px-8">
+      <div class="flex items-center justify-between h-16">
+        <div class="flex items-center">
+          <div class="flex-shrink-0">
+            <img
+              class="block lg:hidden h-8 w-auto"
+              src="@/assets/images/navbar_logo.svg"
+              alt="Workflow"
+            />
+            <img
+              class="hidden lg:block h-8 w-auto"
+              src="@/assets/images/navbar_logo.svg"
+            />
```

Comment:
It's the same piecie of code - do we really need to duplicate the code? Also, the icon will look better, if it has height `h-10`.

Reply:
Code was adjusted